src/charts/c3/c3.mixin.js

```diff
@@ -0,0 +1,57 @@
+angular.module('patternfly.charts').factory('ChartsMixin', function() {
```

Comment:
Should this be a factory?  I feel like you could accomplish the same thing using a constant.  Where is the title set from outside of this factory?